src/popover/popover.js

```diff
@@ -16,4 +16,25 @@ angular.module( 'ui.bootstrap.popover', [ 'ui.bootstrap.tooltip' ] )
 
 .directive( 'popover', [ '$tooltip', function ( $tooltip ) {
   return $tooltip( 'popover', 'popover', 'click' );
+}])
+
+.directive( 'popoverTemplatePopup', [ function () {
+  return {
+    restrict: 'EA',
+    replace: true,
+    scope: { title: '@', template: '=', placement: '@', animation: '&', isOpen: '&' },
```

Comment:
Both this and the other comment truly relates to the current `popover.js` implementation, not exactly to this small effort to make `popover-template` available...

Here, again, I'm just making it behave exactly the same as the normal popover:
https://github.com/angular-ui/bootstrap/blob/master/src/popover/popover.js#L12